src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/WorkflowFaultEvent.Fields.Edit.cshtml

```diff
@@ -7,19 +7,18 @@
      <textarea asp-for="ErrorFilter" hidden>@Html.Raw(Model.ErrorFilter)</textarea>
     <span asp-validation-for="ErrorFilter"></span>
     <span class="hint">@T["A collection of workflow names that need to return a collection of strings. Javascript syntax."]</span><br />
-    <span class="hint">@T["Get error information： input('{0}')",WorkflowFaultModel.WorkflowFaultInputKey]</span><br />
+    <span class="hint">@T["Get error information： input('{0}')", WorkflowFaultModel.WorkflowFaultInputKey]</span><br />
     <span class="hint">@T["Contains the following attributes："]</span><br />
-    <span class="hint">@T["{0}：workflow Id",nameof( WorkflowFaultModel.WorkflowId)]</span><br />
-    <span class="hint">@T["{0}：Workflow name",nameof( WorkflowFaultModel.WorkflowName)]</span><br />
-    <span class="hint">@T["{0}：Number of executed nodes",nameof( WorkflowFaultModel.ExcutedActivityCount)]</span><br />
-    <span class="hint">@T["{0}：The name of the node",nameof( WorkflowFaultModel.ActivityDisplayName)]</span><br />
-    <span class="hint">@T["{0}：Node Type Name",nameof( WorkflowFaultModel.ActivityTypeName)]</span><br />
-    <span class="hint">@T["{0}：Node Id",nameof( WorkflowFaultModel.ActivityId)]</span><br />
-    <span class="hint">@T["{0}：Exception Information Summary",nameof( WorkflowFaultModel.ErrorMessage)]</span><br />
-    <span class="hint">@T["{0}：Exception Details",nameof( WorkflowFaultModel.ExceptionDetails)]</span><br />
-    <span class="hint">@T["{0}：Contains error messages manually raised by workflow nodes",nameof( WorkflowFaultModel.FaultMessage)]</span><br />
-    
-</div>
+    <span class="hint">@T["{0}：workflow Id", nameof( WorkflowFaultModel.WorkflowId)]</span><br />
```

Comment:
Remove extra space after `nameof` in all the places

Reply:
done